validation-test/stdlib/StringUTF8.swift

```diff
@@ -49,6 +50,35 @@ var strings: Array<String> = [
 
 let kCFStringEncodingASCII: UInt32 = 0x0600
 
+#if _runtime(_ObjC)
+
+var utf16ByteSequences = utf16Tests.flatMap { $0.value }.map { $0.encoded }
+
+private func testForeignContiguous(slowString: NSSlowString, string: String) {
+  // Lazily bridged strings are not contiguous UTF-8
+  var slowString = NSSlowString(string: string) as String
+  expectFalse(slowString.isFastUTF8)
+  expectEqualSequence(string.utf8, slowString.utf8)
+  
+  // They become fast when mutated
+  slowString.makeNative()
+  expectTrue(slowString.isFastUTF8)
```

Comment:
Oh right, this test predates contiguous strings proposal. You can go this route, or update the test to use `isContiguousUTF8`, `makeContiguousUTF8()`, and `withUTF8()`.

Reply:
Noted. I'm probably going to leave it as unchanged as possible for now.